Sources/SendGrid/Provider.swift

```diff
@@ -1,6 +1,6 @@
 import Vapor
 
-public final class Provider: Vapor.Provider {
+public final class SendGridProvider: Vapor.Provider {
```

Comment:
I'd rather rename the module and follow the existing patterns [(1)](https://github.com/vapor/leaf-provider/blob/master/Sources/LeafProvider/Provider.swift#L7), [(2)](https://github.com/vapor/auth-provider/blob/master/Sources/AuthProvider/Provider.swift#L5), [(3)](https://github.com/vapor/mysql-provider/blob/master/Sources/MySQLProvider/Provider.swift)